### PR TITLE
Auth Callout - Fix Types

### PR DESCRIPTION
**Pull request by rickdotnet**
Two types in particular prevented me from properly responding to auth callout in non-operator mode. This PR contains the adjustments that enabled a successful response.

### CODE REVIEW

**Review by mtmk — APPROVED** [review]
LGTM thanks @rickdotnet

File: NATS.Jwt/Models/NatsClientInformation.cs [hunk 1]
```diff
@@ -23,7 +23,7 @@ public record NatsClientInformation
     /// </summary>
     [JsonPropertyName("id")]
     [JsonIgnore(Condition = JsonIgnoreCondition.WhenWritingDefault)]
-    public string Id { get; set; }
+    public long Id { get; set; }
```
Comment (mtmk) [inline NATS.Jwt/Models/NatsClientInformation.cs:26]: are you sure about this one? it looks like it needs to represent base32 strings of nkeys public keys

**Review by mtmk — COMMENTED** [review]
question about the id

File: NATS.Jwt/Models/NatsClientInformation.cs (hunk 1 above)
Comment (rickdotnet) [reply NATS.Jwt/Models/NatsClientInformation.cs:26]: Actually, check the [output here](https://gist.github.com/rickdotnet/2c556cb218b1fa2493da4a38c66fad05). The client id is `31` as opposed to `"31"`

```
{
  "aud": "nats-authorization-request",
  "exp": 1723166342,
  "jti": "2KW2GUTFIO4VQMK5QUB5IU7FE2F365MTXYEKKGXCZGMIELMYGEOQ",
  "iat": 1723166340,
  "iss": "NAL2F6RPUKRNRVTXT6XDHGZWUWD54GWZZFIJ5KU5OHEURSVWS7T4GBPZ",
  "sub": "AACILD5SO3CCBAT5JJVOTWLVBLABXLPP2Q76SI5KSUYFV7STPTALPAG2",
  "nats": {
    "server_id": {
      "name": "NAL2F6RPUKRNRVTXT6XDHGZWUWD54GWZZFIJ5KU5OHEURSVWS7T4GBPZ",
      "host": "0.0.0.0",
      "id": "NAL2F6RPUKRNRVTXT6XDHGZWUWD54GWZZFIJ5KU5OHEURSVWS7T4GBPZ",
      "version": "2.10.18"
    },
    "user_nkey": "UCO3JSTTM3ZDZ6QPTED2SM472PHK264ZHWGPU5I4OQD4AWLTWYF5BLIQ",
    "client_info": {
      "host": "::1",
      "id": 31,
      "user": "bob",
      "name": "NATS CLI Version 0.1.4",
      "kind": "Client",
      "type": "nats"
    },
    "connect_opts": {
      "user": "bob",
      "pass": "bob",
      "name": "NATS CLI Version 0.1.4",
      "lang": "go",
      "version": "1.33.1",
      "protocol": 1
    },
    "type": "authorization_request",
    "version": 2
  }
}
```

Comment (rickdotnet) [reply NATS.Jwt/Models/NatsClientInformation.cs:26]: > it looks like it needs to represent base32 strings of nkeys public keys

Where do you see this? I need to update my notes with a good references of these types.

Comment (mtmk) [reply NATS.Jwt/Models/NatsClientInformation.cs]: my mistake thanks for the detail. I copied from here: https://github.com/nats-io/jwt/blob/7b33986cffa3bd7df2277d853249c385edf9315f/v2/authorization_claims.go#L38

**Review by mtmk — APPROVED** [review]
LGTM

**Pull request merged**